web.tf

```diff
@@ -24,7 +24,7 @@ resource "aws_security_group" "acme_web" {
     from_port   = 80
     to_port     = 80
     protocol    = "tcp"
-    cidr_blocks = ["0.0.0.0/0"]
```

Comment:
**Accurics remediation**
 
**[HIGH]: ssh port open to internet**
 
**Vulnerability**
ssh port open to internet
 
**Remediation**
Change cidr_block to non 0.0.0.0/0